spec/index.bs

```diff
@@ -1206,8 +1199,8 @@ empty string instead, and then do as described below:
 1.  Let |value| be the result of calling [$Get Trusted Type compliant string$] with
     {{TrustedScript}}, [=this=]'s [=relevant global object=], the given value, `HTMLScriptElement textContent`, and
     `script`.
-1.  Set [=this=]'s [=script text=] value to |value|.
 1.  Run [=set text content=] with [=this=] and |value|.
+1.  Set [=this=]'s [=HTMLScriptElement/changed by trusted sink=] to true.
```

Comment:
If as you suggest below "prepare the script" runs at the end of "set text content" it seems this wouldn't really work?

Reply:
Ah yeah sorry I forgot to change the order back for this one